scripts/watchGo.js

```
@@ -0,0 +1,160 @@
+
+import watch from 'node-watch';
+import { spawn, exec } from 'child_process';
+import path from 'path';
+import fs from 'fs';
+const dirs = process.argv.slice(2);
```

ESLint

[error] 'process' is not defined.

(no-undef)